src/test/java/org/opensearch/ad/feature/SearchFeatureDaoTests.java

```diff
@@ -124,6 +125,7 @@
 @RunWith(PowerMockRunner.class)
 @PowerMockRunnerDelegate(JUnitParamsRunner.class)
 @PrepareForTest({ ParseUtils.class, Gson.class })
+@Ignore
```

Comment:
I recall reading somewhere that we weren't supposed to use this (thus commenting things out).  Am I misremembering?

Reply:
I tried to comment the tests the same way we did before. @saratvemulapalli any thoughts here?

Reply:
I remember trying it when trying to remove client and getting some sort of gradle check failure.  If it's passing for you with these, you're probably fine.